tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -17,37 +17,47 @@
 use base "y2x11test";
 use strict;
 use testapi;
-use utils qw(type_string_slow clear_console);
+use utils 'clear_console';
+use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
 
 sub run {
-    my $self   = shift;
-    my $module = "host";
-
+    my $self         = shift;
+    my $module       = "host";
+    my $dm           = lc get_var('DESKTOP');
+    my $hosts_params = {
+        ip    => '195.135.221.134',
+        fqdn  => 'download.opensuse.org',
+        alias => 'download-srv'
+    };
+    my %execute = (
+        gnome => \&turn_off_gnome_screensaver,
+        kde   => \&turn_off_kde_screensaver
+    );
+    $execute{$dm}->();
     select_console 'root-console';
     #	add 1 entry to /etc/hosts and edit it later
     script_run "echo '80.92.65.53    n-tv.de ntv' >> /etc/hosts";
     clear_console;
     select_console 'x11';
     $self->launch_yast2_module_x11($module, match_timeout => 90);
     assert_and_click "yast2_hostnames_added";
-    wait_still_screen 1;
-    wait_screen_change { send_key 'alt-i'; };
-    send_key 'alt-t';
-    type_string 'download-srv';
-    wait_still_screen 1;
-    send_key 'alt-h';
-    type_string 'download.opensuse.org';
-    wait_still_screen 1;
     send_key 'alt-i';
-    type_string_slow '195.135.221.134';
-    assert_and_click 'yast2_hostnames_changed_ok';
+    assert_screen 'yast2_hostnames_edit_popup';
+    send_key 'alt-i';
+    type_string($hosts_params->{ip}, max_interval => 13, wait_still_screen => 0.05, timeout => 5, similarity_level => 38);
+    send_key 'tab';
+    type_string($hosts_params->{fqdn}, max_interval => 13, wait_still_screen => 0.05, timeout => 5, similarity_level => 38);
+    send_key 'tab';
+    type_string($hosts_params->{alias}, max_interval => 13, wait_still_screen => 0.05, timeout => 5, similarity_level => 38);
```

Comment:
What's about having some wrapper with given set of parameters, as I guess they were derived empirically?

Reply:
Do you mean to put all empirical values under one data structure? Or do you suggest to somehow wrap them into a subroutine?

Reply:
I guess more like second, so similarly to `type_string_slow` and `type_string_very_slow`. WDYT?

Reply:
So, to resolve this one, what do you think about some wrapper? If you object, I'm fine to leave it as is, but I'm quite sure that this will be handy in many other scenarios.

Reply:
Sorry for the delay. I will deliver the update during the day.

Reply:
are you ok to leave the wrapper for now in the test module code? I am not sure if I have set the values correctly for OSD or O3. I am expecting some minor updates after the code gets merged.

Reply:
Sure, fine with me. Let's merge then ;)

Reply:
ah, I have pushed the updates a few minutes after the PR got merged. ups...

Reply:
I've misunderstood you. But no worries, create new PR ;)

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6981